src/lara-plugin/plugins/embeddable-runtime-context.spec.ts

```diff
@@ -67,7 +67,7 @@ describe("Embeddable runtime context helper", () => {
   describe("#getReportingUrl", () => {
     it("returns null when interactiveStateUrl is not available", () => {
       const runtimeContext = generateEmbeddableRuntimeContext(
-        Object.assign({}, embeddableContext, {interactiveStateUrl: null})
+        { ...embeddableContext, interactiveStateUrl: null}
```

Comment:
Note on this module:
Scott requested that I mark all code additions inside of the `lara-plugin` folder with the following:
`// ACTIVITY_PLAYER_CODE:`
And to comment out instead of removing any existing LARA code and mark the commented code with the following:
`// LARA_CODE`
I believe this is so we can more easily compare the activity player code with the original code in the LARA project and potentially unify the two code bases.